AirCasting/MobilePeripheralSessionManager.swift

```diff
@@ -132,8 +138,8 @@ class MobilePeripheralSessionManager {
         }
     }
 
-    private func updateStreams(stream: ABMeasurementStream, sessionUUID: SessionUUID) throws {
-        let  location = locationProvider.currentLocation?.coordinate
+    private func updateStreams(stream: ABMeasurementStream, sessionUUID: SessionUUID, isLocationTracked: Bool) throws {
+        let  location = isLocationTracked ? locationProvider.currentLocation?.coordinate : CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)
```

Comment:
this `CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)` is something we're using all over the place. Maybe it would be more readable if we moved it to an extension:
```Swift
extension CLLocationCoordinate2D {
    static var nevermind: CLLocationCoordinate2D { .init(latitude: 200.0, longitude: 200.0) }
}
```

This way the callsite would be more readable:
```Swift
let location = isLocationTracked ? getCoordinate() : .nevermind
```

the name is just off the top of my head, can be different ofc

Reply:
Looks cool 😎 Good idea